### PR TITLE
chore(scripts): add deploy bitxhub scripts

### PR DESCRIPTION
**Pull request by Alexader**
add deploy scripts for compiling linux binary and deploying to server

### CODE REVIEW

File: scripts/boot_bitxhub.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  tmux splitw -v -p 50
+  tmux splitw -h -p 50
+  tmux selectp -t 0
+  tmux splitw -h -p 50
+}
+
+function start() {
+  cd ${CURRENT_PATH}
+  rm -rf build
+  tar -xf build.tar.gz
```
Comment (alburtwang) [inline scripts/boot_bitxhub.sh:15]: rename build.tar.gz to bitxhub-version.tar.gz

Comment (Alexader) [reply scripts/boot_bitxhub.sh:15]: pack this with dynamic version and commit hash now.

File: scripts/deploy.sh [hunk 2]
```diff
@@ -0,0 +1,72 @@
+set -e
+
+source x.sh
+
+CURRENT_PATH=$(pwd)
+PROJECT_PATH=$(dirname "${CURRENT_PATH}")
+BUILD_PATH=${CURRENT_PATH}/build
+GIT_COMMIT=$(git log --pretty=format:'%h' -n 1)
+APP_VERSION=1.0.0-rc1-${GIT_COMMIT}
```
Comment (4ever9) [inline scripts/deploy.sh:9]: It is recommended to get the latest version rather than a fixed name

File: scripts/deploy.sh [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+USERNAME=hyperchain
+SERVER_BUILD_PATH="/home/hyperchain/bitxhub"
```
Comment (4ever9) [inline scripts/deploy.sh:12]: `USERNAME` and `SERVER_BUILD_PATH` are better passed in as argumants

File: scripts/boot_bitxhub.sh [hunk 4]
```diff
@@ -0,0 +1,39 @@
+CURRENT_PATH=$(pwd)
```
Comment (4ever9) [inline scripts/boot_bitxhub.sh:1]: This file should be merged with `deploy.sh`

File: scripts/deploy.sh [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+  ssh -t ${USERNAME}@"${BITXHUB_ADDR}" '
+    cd '${SERVER_BUILD_PATH}'
+    bash boot_bitxhub.sh 4
+    tmux attach-session -t bitxhub
+'
+}
+
+MODE=$1
+
+if [ "$MODE" == "deploy" ]; then
```
Comment (4ever9) [inline scripts/deploy.sh:66]: It`s redundant......

**Pull request merged**